src/CloudWatchLogs/CloudWatchLogsClient.php

```diff
@@ -155,4 +156,61 @@
  * @method \Aws\Result updateLogAnomalyDetector(array $args = [])
  * @method \GuzzleHttp\Promise\Promise updateLogAnomalyDetectorAsync(array $args = [])
  */
-class CloudWatchLogsClient extends AwsClient {}
+class CloudWatchLogsClient extends AwsClient {
+    static $streamingCommands = [
```

Comment:
Nit: maybe rename this to `$streamingFlagCommands`?

Reply:
I do not think so. The intention of this variable is to hold the name of the commands that require streaming, therefore I think $streamingCommands suits better here.